app/controllers/v0/benefits_documents_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'lighthouse/benefits_documents/service'
+
+module V0
+  class BenefitsDocumentsController < ApplicationController
+    before_action { authorize :lighthouse, :access? }
+    Raven.tags_context(team: 'benefits-claim-appeal-status', feature: 'benefits-documents')
+
+    def file_number
```

Comment:
Is this an upcoming route? I'm not sure I know why this is here.

Reply:
Oh good catch, thanks Kris. This has since been moved to the service, forgot to remove it here.